src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js

```diff
@@ -18,14 +18,14 @@ import { useCustomInfiniteQuery } from 'src/modules/common/hooks';
 
 export const useTokensTopLskBalance = ({ config: customConfig = {}, options } = {}) => {
```

Comment:
```suggestion
export const useTokenBalancesTop = ({ config: customConfig = {}, options } = {}) => {
```

Reply:
Please rename the file as well `src/modules/token/fungible/hooks/queries/useTokensTopLskBalance.js`

Reply:
Done